cfme/common/provider.py

```diff
@@ -707,18 +645,24 @@ def load_all_provider_templates(self):
             return True
 
 
-@fill.method((Form, BaseProvider.Credential))
+@fill.method((Form, cfme.base.credential.Credential))  # default credential
+@fill.method((Form, cfme.base.credential.AMQPCredential))
+@fill.method((Form, cfme.base.credential.CANDUCredential))
+@fill.method((Form, cfme.base.credential.AzureCredential))
+@fill.method((Form, cfme.base.credential.SSHCredential))
+@fill.method((Form, cfme.base.credential.TokenCredential))
+@fill.method((Form, cfme.base.credential.ServiceAccountCredential))
```

Comment:
I'll remove this fill method when finish providers' conversion to widgetastic. All providers will have their own fill statements based on endpoints. I just can't include all the planned changes to one PR.